ci/tests/mgmtcenter/mgmt_center_test.py

```diff
@@ -15,13 +15,16 @@
 from ci.tests.general import general
 from ci.tests.mgmtcenter import generic
 from ci import autotests
+from nose.plugins.skip import SkipTest
 from ci.tests.general.connection import Connection
 
 testsToRun = general.get_tests_to_run(autotests.get_test_level())
+DEVSTACK_INSTALLED = generic.is_devstack_installed()
 
 
 def setup():
-    generic.create_generic_mgmt_center()
+    if DEVSTACK_INSTALLED:
+        generic.create_generic_mgmt_center()
```

Comment:
I'd put the check 'is_devstack_installed' in the create_generic_mgmt_center to keep things centralized